api/v1alpha1/oidc_types.go

```diff
@@ -36,20 +36,21 @@ type OIDC struct {
 	// specified.
 	// +optional
 	Scopes []string `json:"scopes,omitempty"`
+
+	// The redirect URL to be used in the OIDC
+	// [Authentication Request](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest).
+	// If not specified, uses the default redirect URI "%REQ(x-forwarded-proto)%://%REQ(:authority)%/oauth2/callback"
+	RedirectURL *string `json:"redirectURL,omitempty"`
+
+	// The path to sign a user out, clearing their credential cookies.
+	// If not specified, uses a default signout path "/signout"
+	SignoutPath *string `json:"signoutPath,omitempty"`
```

Comment:
this should be `logout`

should this be `logoutPath` or `logoutUri` ?

Reply:
It should be path because the schem and port are not included. For example `/petstore/logout`

Reply:
envoy docs calls its signout but the oidc docs calls it `logout`, so prefer `logoutPath`